<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2025-06-11T11:41:11 IST" name="Default suite" finished-at="2025-06-11T11:41:27 IST" duration-ms="15308">
    <groups>
    </groups>
    <test started-at="2025-06-11T11:41:11 IST" name="Default test" finished-at="2025-06-11T11:41:27 IST" duration-ms="15308">
      <class name="com.comcast.crm.orgtest.PracticeTest">
        <test-method is-config="true" signature="configBeforeSuite()[pri:0, instance:com.comcast.crm.orgtest.PracticeTest@402c4085]" started-at="2025-06-11T11:41:11 IST" name="configBeforeSuite" finished-at="2025-06-11T11:41:11 IST" duration-ms="44" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBeforeSuite -->
        <test-method is-config="true" signature="configBeforeClass()[pri:0, instance:com.comcast.crm.orgtest.PracticeTest@402c4085]" started-at="2025-06-11T11:41:11 IST" name="configBeforeClass" finished-at="2025-06-11T11:41:16 IST" duration-ms="4992" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configBeforeClass -->
        <test-method is-config="true" signature="login()[pri:0, instance:com.comcast.crm.orgtest.PracticeTest@402c4085]" started-at="2025-06-11T11:41:16 IST" name="login" finished-at="2025-06-11T11:41:21 IST" duration-ms="4286" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method signature="prTest()[pri:0, instance:com.comcast.crm.orgtest.PracticeTest@402c4085]" started-at="2025-06-11T11:41:21 IST" name="prTest" finished-at="2025-06-11T11:41:21 IST" duration-ms="71" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [vivek] but found [VIVEK]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [vivek] but found [VIVEK]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at com.comcast.crm.orgtest.PracticeTest.prTest(PracticeTest.java:12)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- prTest -->
        <test-method is-config="true" signature="logout()[pri:0, instance:com.comcast.crm.orgtest.PracticeTest@402c4085]" started-at="2025-06-11T11:41:21 IST" name="logout" finished-at="2025-06-11T11:41:25 IST" duration-ms="4653" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- logout -->
        <test-method is-config="true" signature="configAfterClass()[pri:0, instance:com.comcast.crm.orgtest.PracticeTest@402c4085]" started-at="2025-06-11T11:41:25 IST" name="configAfterClass" finished-at="2025-06-11T11:41:27 IST" duration-ms="1129" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- configAfterClass -->
        <test-method is-config="true" signature="toCloseDB()[pri:0, instance:com.comcast.crm.orgtest.PracticeTest@402c4085]" started-at="2025-06-11T11:41:27 IST" name="toCloseDB" finished-at="2025-06-11T11:41:27 IST" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- toCloseDB -->
      </class> <!-- com.comcast.crm.orgtest.PracticeTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
